src/components/FixedGrid/FixedGridColumn.styles.ts

```diff
@@ -0,0 +1,40 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { mergeStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+export interface FixedGridColumnStyleProps {
+  fixed?: boolean;
+  width?: number;
+  verticalAlign: 'top' | 'middle' | 'bottom';
+}
+
+const alignItems: { [i in 'top' | 'middle' | 'bottom']: 'flex-start' | 'flex-end' | 'center' } = {
```

Comment:
TypeScript is now smart enough map enum values to object keys directly. See here:
https://github.com/Microsoft/YamUI/pull/380/files#diff-633c2868943735bc3166c3960a5bcd29R13

and usage here:
https://github.com/Microsoft/YamUI/pull/380/files#diff-633c2868943735bc3166c3960a5bcd29R29


does this not work for the string literal union types?

Reply:
The trouble here was that I need the values to be typed. `alignItems` can't be a string.  Do you know if it's possible to type the values without typing the keys?